src/bentoml_cli/serve.py

```diff
@@ -192,7 +193,12 @@ def cli():
         show_default=True,
         hidden=True,
     )
-    @env_manager
+    @click.option(
+        "--env",
+        type=click.STRING,
+        help="List of environment variables pass by --env key[=value] --env ...",
```

Comment:
It's less useful proving such option in serve command, users are always able to achieve this by setting env in their terminal. Can we just leave this out for this PR?